stories/topics/ref-aap-aws-uninstall.adoc

```diff
@@ -0,0 +1,35 @@
+[id="ref-aws-uninstall"]
+
+= Uninstall
+
+The following document guides how to uninstall {PlatformNameShort} from {Market}. If extension nodes were deployed, a prerequisite step must be taken to remove the extension nodes before uninstalling the stack.
+
+== Uninstall extension nodes
+
+If extension nodes were installed, follow these steps to remove them before continuing with the uninstall. If extension nodes were not installed, skip these steps.
```

Comment:
If you move the 'If you deployed extension nodes...' sentence from above to here, you can just add 'If you did not install extension nodes, skip theses steps.'

Reply:
Fixed!